packages/phone-number-privacy/package.json

```diff
@@ -27,7 +27,8 @@
     "pg": "^7.18.2"
   },
   "devDependencies": {
-    "firebase-functions-test": "^0.1.6"
+    "firebase-functions-test": "^0.1.6",
+    "jest": "^25.4.0"
```

Comment:
we already pull in jest in the collective package.json -- are you just including it here for the version bump?

Reply:
Having a project level jest was necessary for the local debugging to work (see launch.json comment).

Reply:
Will move to the root of monorepo as part of the launch.json migration in a future PR (removing here for now).